src/CMakeLists.txt

```diff
@@ -69,6 +69,16 @@ else()
   )
 endif()
 
+option(FORCE_EMBEDDED_MINIZIP "Use embedded minizip sources" OFF)
+if(NOT FORCE_EMBEDDED_MINIZIP)
```

Comment:
Try to mimic existing style for cmake options: 
- define option at beginning of top level CMakeLists.txt
- name it `OPTION_...`
- add it to the table in Readme

Reply:
My comment was probably unclear. I think best to leave it in permanent.